lib/plugins/inventory.js

```diff
@@ -22,10 +22,12 @@ function inject (bot, { version, hideErrors }) {
     bobberId = mcData.entitiesByName.fishing_bobber.id
   }
 
+  const inventoryEvents = new EventEmitter()
```

Comment:
You can share this eventemitter with digging.js

I'd name it something like `bot._internalEvents`